build/common.mk

```diff
@@ -171,14 +204,13 @@ install::
 	cp yarn.lock "$(PULUMI_NODE_MODULES)/$(NODE_MODULE_NAME)"
 	rm -rf "$(PULUMI_NODE_MODULES)/$(NODE_MODULE_NAME)/node_modules"
 	cd "$(PULUMI_NODE_MODULES)/$(NODE_MODULE_NAME)" && \
-		yarn install --offline --production && \
-		(yarn unlink > /dev/null 2>&1 || true) && \
-		yarn link
+	yarn install --offline --production && \
+	(yarn unlink > /dev/null 2>&1 || true) && \
+	yarn link
 endif
 
 only_build:: build install
 only_test:: lint test_all
-only_test_fast:: lint test_fast
```

Comment:
i think these removals aren't actually appropriate.  i was actually just adding them to awsinfra to match pulumi/pulumi

Reply:
@ellismg i can see we're inconsistent across our repos here.  what's teh right pattern we should be following/